server/controllers/user-controller-v4.js

```diff
@@ -0,0 +1,360 @@
+import bcrypt from 'bcrypt';
+import {
+  UserDetails,
+  Memberships,
+} from '../models';
+import JwTokens from '../middleware/helpers';
+import CheckSession from '../middleware/session';
+import UserHelper from '../helpers/user-signup';
```

Comment:
I can see the file name extension is in .jsx. It is preferable to make this a .js file and also use camel casing for the file name e.g `userSignUp` or in line with what this module does, `inputValidator` would suffice.